.travis.yml

```diff
@@ -6,14 +6,17 @@ php:
   - 5.5
   - 5.6
   - hhvm
+  - 7
 
 env:
-  - VARNISH_VERSION=3.0
+  - VARNISH_VERSION=4.0
 
 matrix:
   include:
     - php: 5.5
-      env: VARNISH_VERSION=4.0
+      env: VARNISH_VERSION=3.0
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
It seems `env` overrides the global `env` completely, so we need another `VARNISH_VERSION=4.0` (or `3.0`) here.